x-pack/plugins/apm/server/routes/errors.ts

```diff
@@ -31,7 +31,8 @@ export function initErrorsApi(server: Server) {
           sortField: Joi.string(),
           sortDirection: Joi.string()
         })
-      }
+      },
+      tags: ['access:apm']
```

Comment:
`tags` is a Hapi thing or something we built and extended for access control?

Reply:
This is something that Hapi lets us configure on the route itself, which we're taking advantage of to block access to specific routes. This wasn't the original intent for tags in Hapi, and we'll have more options when we switch to the new platform when plugins aren't directly interacting with the hapi server. I opened up https://github.com/elastic/kibana/pull/32915 to rework this syntax slightly, and also recently tried another hapi specific approach, but I'm open to alternatives.